package.json

```diff
@@ -1155,6 +1155,18 @@
 										"type": "command",
 										"command": "editor.action.format"
 									},
+									{
+										"key": "C",
+										"name": "Jump to next change",
+										"type": "command",
+										"command": "workbench.action.editor.nextChange"
+									},
```

Comment:
We sort the binding key by lower case then upper case. So this binding should sit right above the key `I`